app/dashboard/invoices/[invoicesId]/page.tsx

```
@@ -97,21 +136,21 @@ export default async function InvoicesIdPage({
                                     Created by
                                     </div>
                                     <div className="flex-1 text-white">
-                                    <span className="bg-gray-700 text-xs px-2 py-1 rounded">System</span>
+                                        <span className="bg-gray-700 text-xs px-2 py-1 rounded">{(session.user && session.user.email) ? session.user.email.split('@')[0] : 'Unknown User'}</span>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)